cmd/ddev/cmd/config.go

```diff
@@ -35,6 +52,12 @@ var ConfigCommand = &cobra.Command{
 		err = c.Write()
 		if err != nil {
 			util.Failed("Could not write ddev config file: %v\n", err)
+
+		}
+
+		// If a provider is specified, prompt about whether to do an import after config.
+		if provider != "" {
+			util.Success("Configuration complete. You may now run `ddev start` or `ddev pull`")
```

Comment:
This should not have backticks. Sorry, a nit, but this output is not markdown. Please check all of these to remove the backticks, which are just distracting to a normal user. Quotes would be OK, like "you may now run "ddev start" or "ddev pull"